docs/sources/packages_api/runtime/backendsrv.md

```diff
@@ -26,19 +27,21 @@ import { BackendSrv } from '@grafana/runtime';
 
 ## Remarks
 
-By default Grafana will display an error message alert if the remote call fails. If you want to prevent this from happending you need to catch the error thrown by the BackendSrv and set the `isHandled = true` on the incoming error.
+By default Grafana will display an error message alert if the remote call fails. If you want to prevent this from happending you need to catch the error thrown by the BackendSrv and set the `showErrorAlert = true` on the request options object.

```

Comment:
```suggestion
By default, Grafana displays an error message alert if the remote call fails. To prevent this from happening, you need to catch the error thrown by the BackendSrv and set the `showErrorAlert = true` on the request options object.

```